libs/components/src/lib/data-grid/data-grid-cell.scss

```
@@ -61,3 +62,9 @@ slot {
 	text-overflow: ellipsis;
 	white-space: nowrap;
 }
+
+.sort-icons-wrapper {
+	display: flex;
+	flex-direction: column;
```

```suggestion
	flex-direction: column;
	margin-inline-start: auto;
```
this is for sticking it to the right